WordPress/src/main/java/org/wordpress/android/ui/accounts/NewUserFragment.java

```diff
@@ -689,6 +689,7 @@ public void onNewSiteCreated(OnNewSiteCreated event) {
             mDispatcher.dispatch(AccountActionBuilder.newCreateNewAccountAction(mNewAccountPayload));
             return;
         }
+        AnalyticsTracker.track(AnalyticsTracker.Stat.CREATED_SITE);
```

Comment:
Eventually yes. But the idea of this patch is to add back analytics events removed during FluxC integration and track the same events at the same place (when it's possible).